components/scream/src/share/io/tests/io.cpp

```diff
@@ -143,6 +143,15 @@ TEST_CASE("input_output_basic","io")
   auto max_params = get_in_params("Max",io_comm,time);
   auto multi_params = get_in_params("Multisnap",io_comm,t0+dt);
   Real tol = 100*std::numeric_limits<Real>::epsilon();
+  // Test that pio_inq_dimlen is correct, using a file from one of the above parameter lists.
```

Comment:
TODO: we should have a small nc dummy file in the repo, and test all input functions in a separate unit test.